test/MockCommandWithAlias.ts

```diff
@@ -0,0 +1,17 @@
+import Command from "../src/abstracts/Command";
+import { Message } from "discord.js";
+
+export default class MockCommandWithAlias extends Command {
+	constructor() {
+		super(
+			"mock-alias",
+			"Mock Command With Alias",
+			{
+				aliases: ["mocky", "mockster"]
+			});
+	}
```

Comment:
Indentation seems a bit messed up here.
```suggestion
	constructor() {
		super(
			"mock-alias",
			"Mock Command With Alias",
			{
				aliases: ["mocky", "mockster"]
			}
		);
	}
```